08-regression2.Rmd

```diff
@@ -319,8 +341,8 @@ set. On such a data set, K-NN regression may fare better. Additionally, there
 are other types of regression you can learn about in future courses that may do
 even better at predicting with such data.
 
-How do these two models compare on the Sacramento house prices data set? On 
-the visualizations above we also printed the RMPSE as calculated from 
+How do these two models compare on the Sacramento house prices data set? In
+Figure \@ref(fig:08-compareRegression), we also printed the RMPSE as calculated from 
 predicting on the test data set that was not used to train/fit the models. The RMPSE for the simple linear
```

Comment:
I don't think we should compare the test errors here - that is sending the wrong message and encouraging bad behaviour. We should be comparing cross-validated error. We can still get a cross validated error on linear regression, even if we are not need to tune a parameter.